src/Blazor.Gitter.Client/Styles/_chatroom.scss

```diff
@@ -132,9 +132,12 @@
     font-size: $font-size-small;
 }
 
-.chat-room-messages {}
+.chat-room-messages {
+    overflow-y: auto;

```

Comment:
This is so we can monitor scolling from within the RoomMessages component - previously the scroll moved to the Room level, which meant we lost scope.